src/main/java/bio/terra/flight/dataset/create/DatasetCreateFlight.java

```diff
@@ -26,5 +26,6 @@ public DatasetCreateFlight(FlightMap inputParameters, Object applicationContext)
         addStep(new CreateDatasetMetadataStep(datasetDao, datasetService));
         addStep(new CreateDatasetPrimaryDataStep(bigQueryPdao, datasetService, datasetDao, fileDao));
         addStep(new CreateDatasetAuthzResource(samClient));
+        addStep(new SetBigQueryAuthorization(bigQueryPdao));
```

Comment:
Could we put the primary data step and the authorization step into one step? That would save the trouble of holding all of the state in the BigQueryContainer object between the steps.

Reply:
yes!